framework/src/application/node/forger/forger.ts

```diff
@@ -343,12 +345,12 @@ export class Forger {
 	}
 
 	public async forge(): Promise<void> {
+		const MS_IN_A_SEC = 1000;
 		const currentSlot = this._chainModule.slots.getSlotNumber();
-		const currentSlotTime = this._chainModule.slots.getRealTime(
-			this._chainModule.slots.getSlotTime(currentSlot),
-		);
 
-		const currentTime = new Date().getTime();
+		const currentSlotTime = this._chainModule.slots.getSlotTime(currentSlot);
+
+		const currentTime = Math.floor(new Date().getTime() / MS_IN_A_SEC);
```

Comment:
Output of `getSlotTime(currentSlot)` is in Unix timestamp (meant multiplied by 1000). So you don't need to divide `currentTime` here.